### DISCUSSION TITLE
Any tools to calculate "coverage" of CODEOWNERS?

### DISCUSSION THREAD

**Discussion by wtait1-ff** [community/community#36200] [Other Feature Feedback, Questions, & Ideas]
I'm wondering if anyone is aware of any open source tooling that can give a "coverage" report of the CODEOWNERS file. This would help as we slowly introduce CODEOWNERS rules across our monorepo, since we could build automation to check that our coverage never regresses.

**Comment by ossanna16** [comment]
Hey there @wtait1-ff and thank you for asking a good question! I'm glad you found our community 🙂
